src/little-bang-ast/little_bang_ast.ml

```diff
@@ -20,8 +20,7 @@ let new_fresh_ident () =
       "Tiny_bang_ast.new_fresh_ident returned Ident (not Fresh_ident)"
   | Tiny_bang_ast.Builtin_ident _ ->
     raise @@ Invariant_failure
-      "Tiny_bang_ast.new_fresh_ident returned Builtin_ident (not Fresh_ident)"      
-;;
```

Comment:
We're trying to follow a standard of always using `;;` where it makes sense. Apparently, this helps OCaml give better error messages. Is there a reason why you removed the `;;` from here?